spec/support/auth_helper.rb

```diff
@@ -1,7 +1,8 @@
 module AuthHelper
   def http_login
-    username = 'user'
-    password = 'password'
+    username = ENV['ADMIN_NAME']
```

Comment:
Use fetch here so that the test suite fails if the environment variables are missing